elliottlib/bzutil.py

```diff
@@ -58,6 +58,8 @@ def whiteboard_component(self):
         raise NotImplementedError
 
     def is_tracker_bug(self):
+        if not set(constants.TRACKER_BUG_KEYWORDS).issubset(set(self.keywords)) and re.search(r'^CVE-(\d+)-(\d+)', self.summary):
```

Comment:
@Ximinhan Let's not have this logic in this method. I propose to have it in a separate function, 
```python
def looks_like_tracker_bug(self):
    return re.search(r'^CVE-(\d+)-(\d+)', self.summary)
```